docker/keycloak/README.md

```diff
@@ -52,6 +52,7 @@ VCR.py). The primary reason this setup exists, is for automated testing reasons.
   attributes from `eherkenning-bewindvoering`.
 - `admin` / `admin`, intended to create as django user (can be made staff). The email address is
   `admin@example.com`.
+- `employee` / `employee`, has the `employee_id` attribute (for org-oidc)
```

Comment:
this isn't necessary, since you can populate the user model field from already existing claims, see the documentation: https://open-forms.readthedocs.io/en/stable/configuration/authentication/oidc_org.html#configuring-oidc-for-login-of-organization-members

In these cases I would probably just map the `sub` or `username` or `email` claim to the `employee_id` field.

Reply:
But isn't this needed for the login itself?

Reply:
you can use the `admin/admin` user for that :)